### PR TITLE
some mappings ^-^

### PR DESCRIPTION
**Pull request by IThundxr**

### CODE REVIEW

File: mappings/net/minecraft/server/WorldStem.mapping [hunk 1]
```diff
@@ -2,7 +2,9 @@ CLASS net/minecraft/unmapped/C_tljyhzqp net/minecraft/server/WorldStem
 	FIELD f_hmwuaxvg saveProperties Lnet/minecraft/unmapped/C_jkevluio;
 	FIELD f_sphgqqzg resourceManager Lnet/minecraft/unmapped/C_carvbxzj;
 	FIELD f_twvealqc resources Lnet/minecraft/unmapped/C_migzkpst;
+	FIELD f_yskbtpkd registries Lnet/minecraft/unmapped/C_bcpxdrik;
 	METHOD close close ()V
 	METHOD m_pfrxpzhv resources ()Lnet/minecraft/unmapped/C_migzkpst;
+	METHOD m_qvpzrpcg registries ()Lnet/minecraft/unmapped/C_bcpxdrik;
```
Comment (supersaiyansubtlety) [inline mappings/net/minecraft/server/WorldStem.mapping:8]:
```suggestion
	METHOD m_qvpzrpcg getRegistries ()Lnet/minecraft/unmapped/C_bcpxdrik;
```
[CONVENTIONS.md "getters"](https://github.com/QuiltMC/quilt-mappings/blob/1.20-pre6/CONVENTIONS.md#getters-setters-withers-and-creators)
> Use "get" for non-boolean getters and other methods that calculate some property with no side effects other than caching a value in a private field.

File: mappings/net/minecraft/server/WorldStem.mapping [hunk 2]
```diff
@@ -2,7 +2,9 @@ CLASS net/minecraft/unmapped/C_tljyhzqp net/minecraft/server/WorldStem
 	FIELD f_hmwuaxvg saveProperties Lnet/minecraft/unmapped/C_jkevluio;
 	FIELD f_sphgqqzg resourceManager Lnet/minecraft/unmapped/C_carvbxzj;
 	FIELD f_twvealqc resources Lnet/minecraft/unmapped/C_migzkpst;
+	FIELD f_yskbtpkd registries Lnet/minecraft/unmapped/C_bcpxdrik;
 	METHOD close close ()V
 	METHOD m_pfrxpzhv resources ()Lnet/minecraft/unmapped/C_migzkpst;
```
Comment (supersaiyansubtlety) [inline mappings/net/minecraft/server/WorldStem.mapping:7]:
```suggestion
	METHOD m_pfrxpzhv getResources ()Lnet/minecraft/unmapped/C_migzkpst;
```
if fixing `registries`, might as well fix this

File: mappings/net/minecraft/server/WorldStem.mapping [hunk 3]
```diff
@@ -2,7 +2,9 @@ CLASS net/minecraft/unmapped/C_tljyhzqp net/minecraft/server/WorldStem
...
+	METHOD m_qvpzrpcg registries ()Lnet/minecraft/unmapped/C_bcpxdrik;
 	METHOD m_vmpnykme resourceManager ()Lnet/minecraft/unmapped/C_carvbxzj;
```
Comment (supersaiyansubtlety) [inline mappings/net/minecraft/server/WorldStem.mapping:9]:
```suggestion
	METHOD m_vmpnykme getResourceManager ()Lnet/minecraft/unmapped/C_carvbxzj;
```
if fixing `registries`, might as well fix this

File: mappings/net/minecraft/server/WorldStem.mapping [hunk 4]
```diff
@@ -2,7 +2,9 @@ CLASS net/minecraft/unmapped/C_tljyhzqp net/minecraft/server/WorldStem
...
 	METHOD m_vqetafdc saveProperties ()Lnet/minecraft/unmapped/C_jkevluio;
```
Comment (supersaiyansubtlety) [inline mappings/net/minecraft/server/WorldStem.mapping:10]:
```suggestion
	METHOD m_vqetafdc getSaveProperties ()Lnet/minecraft/unmapped/C_jkevluio;
```
if fixing `registries`, might as well fix this

**Review by supersaiyansubtlety — CHANGES_REQUESTED** [review]

**Review by supersaiyansubtlety — APPROVED** [review]

**Review by supersaiyansubtlety — APPROVED** [review]

File: mappings/net/minecraft/client/gui/components/debugchart/AbstractDebugChart.mapping [hunk 5]
```diff
@@ -0,0 +1 @@
+CLASS net/minecraft/unmapped/C_bkzzefxr net/minecraft/client/gui/components/debugchart/AbstractDebugChart
```
Comment (eristhea) [inline mappings/net/minecraft/client/gui/components/debugchart/AbstractDebugChart.mapping]: so --
I haven't looked deep in here, but I assume it's stuff like the f3 pie, which would have me place it in `gui/hud` instead of `/components/debugchart`, which creates two brand new packages

File: mappings/com/mojang/realmsclient/RealmsAvailability.mapping [hunk 6]
```diff
@@ -0,0 +1 @@
+CLASS net/minecraft/unmapped/C_aaohlqkp com/mojang/realmsclient/RealmsAvailability
```
Comment (eristhea) [inline mappings/com/mojang/realmsclient/RealmsAvailability.mapping]: `net/minecraft/realms` please! we should probably look into matching the mojang packages here, but we're not going to throw one class in a brand new place for this pr

File: mappings/net/minecraft/advancement/AdvancementVisibilityEvaluator.mapping [hunk 7]
```diff
@@ -2,3 +2,4 @@ CLASS net/minecraft/unmapped/C_hrhneqyx net/minecraft/advancement/AdvancementVis
 	METHOD m_exegzdua (Lnet/minecraft/unmapped/C_kdwyuhdb;Z)Lnet/minecraft/unmapped/C_hrhneqyx$C_bfuhkkvt;
 		ARG 0 advancement
 		ARG 1 alwaysShowIfPossible
+	METHOD m_jmvvzccl evaluateVisibility (Lnet/minecraft/unmapped/C_kdwyuhdb;Ljava/util/function/Predicate;Lnet/minecraft/unmapped/C_hrhneqyx$C_laxhphom;)V
```
Comment (eristhea) [inline mappings/net/minecraft/advancement/AdvancementVisibilityEvaluator.mapping]:
```suggestion
	METHOD m_jmvvzccl evaluate (Lnet/minecraft/unmapped/C_kdwyuhdb;Ljava/util/function/Predicate;Lnet/minecraft/unmapped/C_hrhneqyx$C_laxhphom;)V
```
`visibility` feels redundant

File: mappings/net/minecraft/block/BarrierBlock.mapping [hunk 8]
```diff
@@ -1 +1,2 @@
 CLASS net/minecraft/unmapped/C_flqucikz net/minecraft/block/BarrierBlock
+	FIELD f_rjhoxnad WATER_LOGGED Lnet/minecraft/unmapped/C_xhwijdsd;
```
Comment (eristhea) [inline mappings/net/minecraft/block/BarrierBlock.mapping]:
```suggestion
	FIELD f_rjhoxnad WATERLOGGED Lnet/minecraft/unmapped/C_xhwijdsd;
```
the only uses of waterlogged in the current codebase use it as one word. this also means you'll have to add it to the dictionary in `buildSrc` resources!

File: mappings/net/minecraft/server/WorldStem.mapping [hunk 9]
```diff
@@ -2,7 +2,9 @@ CLASS net/minecraft/unmapped/C_tljyhzqp net/minecraft/server/WorldStem
 	FIELD f_hmwuaxvg saveProperties Lnet/minecraft/unmapped/C_jkevluio;
 	FIELD f_sphgqqzg resourceManager Lnet/minecraft/unmapped/C_carvbxzj;
 	FIELD f_twvealqc resources Lnet/minecraft/unmapped/C_migzkpst;
+	FIELD f_yskbtpkd registries Lnet/minecraft/unmapped/C_bcpxdrik;
 	METHOD close close ()V
-	METHOD m_pfrxpzhv resources ()Lnet/minecraft/unmapped/C_migzkpst;
-	METHOD m_vmpnykme resourceManager ()Lnet/minecraft/unmapped/C_carvbxzj;
-	METHOD m_vqetafdc saveProperties ()Lnet/minecraft/unmapped/C_jkevluio;
+	METHOD m_pfrxpzhv getResources ()Lnet/minecraft/unmapped/C_migzkpst;
```
Comment (eristhea) [inline mappings/net/minecraft/server/WorldStem.mapping]: this is a record, so we don't use the `get` name. please revert the changes to this class!

File: mappings/net/minecraft/server/network/ConfigurationTask.mapping [hunk 10]
```diff
@@ -0,0 +1 @@
+CLASS net/minecraft/unmapped/C_bztbfoyh net/minecraft/server/network/ConfigurationTask
```
Comment (eristhea) [inline mappings/net/minecraft/server/network/ConfigurationTask.mapping]: I prefer the names in #483 for this, please drop!

File: mappings/net/minecraft/server/network/config/JoinWorldTask.mapping [hunk 11]
```diff
@@ -0,0 +1 @@
+CLASS net/minecraft/unmapped/C_yvdbpilm net/minecraft/server/network/config/JoinWorldTask
```
Comment (eristhea) [inline mappings/net/minecraft/server/network/config/JoinWorldTask.mapping]: I prefer the names in #483 for this, please drop!

File: mappings/net/minecraft/world/entity/ai/behavior/declarative/MemoryCondition.mapping [hunk 12]
```diff
@@ -0,0 +1 @@
+CLASS net/minecraft/unmapped/C_brqsauyi net/minecraft/world/entity/ai/behavior/declarative/MemoryCondition
```
Comment (eristhea) [inline mappings/net/minecraft/world/entity/ai/behavior/declarative/MemoryCondition.mapping]: this `ai/behaviour/declarative` package doesn't exist, I'd move this back into `ai/brain`

**Review by eristhea — CHANGES_REQUESTED** [review]

File: mappings/net/minecraft/server/WorldStem.mapping (hunk 1 above)
Comment (eristhea) [reply mappings/net/minecraft/server/WorldStem.mapping:8]: this convention doesn't apply to records, since it's superceded by the convention of the java names

File: mappings/net/minecraft/server/WorldStem.mapping (hunk 9 above)
Comment (supersaiyansubtlety) [reply mappings/net/minecraft/server/WorldStem.mapping]: oh oops, my bad

**Review by eristhea — CHANGES_REQUESTED** [review]
you've applied the proper renames to the mapping files, but you haven't moved the files themselves. this is done automagically in enigma!

**Review by eristhea — APPROVED** [review]
